### PR TITLE
New Guide: Pingdom Uptime Check

### PR DESCRIPTION
**Pull request by alexfornuto**
Supersedes #2363 (Internal PRs get CI tests and test environments)

## Effect
PR includes the following changes:
- Adds Pingdom Uptime Check integration guide
- Includes inline images

## Remaining Work
- [x] Formatting Review
- [x] Technical Review
- [x] Copy Review

### CODE REVIEW

**Comment by sukottokun** [comment]
+1 Thanks, Alex!

File: source/_docs/guides/pingdom-uptime-check.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+3.  In the modal that opens, add the following information where indicated:
+
+    - **Name of Check:** An easily recognizable name for the check, e.g. “mysite.com Home Page.”
+
+    - **Check Interval:** How often Pingdom will check the site. While one minute is the minimum and acceptable to use, often a check every 5 minutes or more is adequate, and will allow for brief network throughput issues. Remember, these checks also will appear in web access logs, so too many checks may make it harder to debug other issues.
+
+    - **Check Type:** You can monitor several different things with Pingdom: email services, network components such as DNS or specific ports, or a website. Choose **Web**.
+
+    - **URL/IP:** On the **Required** tab, enter the URL for the website. If your site is using HTTPS, select that dropdown option. From the **Optional** tab, you can add user credentials and expected response text. This is very useful if you are using Varnish to cache a site; you can create a simple PHP script which queries the database and returns a specific value to determine if the site is functioning as expected.
```
Comment (alexfornuto) [inline source/_docs/guides/pingdom-uptime-check.md]: Perhaps something like:
> If you have a custom domain, we suggest using it here to ensure that all redirects are working. If you're only concerned with the uptime of the site/CMS itself, you can point to your pantheonsite.io domain.

Needs work, but if you wanna add something like that I'm all for it.

Comment (rachelwhitton) [inline source/_docs/guides/pingdom-uptime-check.md]: On second thought, it might be simplest to stay vague.

Comment (alexfornuto) [inline source/_docs/guides/pingdom-uptime-check.md]: Also fine by me.

**Comment by alexfornuto** [comment]
Thanks @sukottokun for the excellent guide, and @rachelwhitton for the thorough review!

**Pull request merged**